drop-transfer/src/storage_dispatch.rs

```diff
@@ -295,8 +269,7 @@ impl From<&crate::Event> for Option<Event> {
                 file_id: file_id.to_string(),
                 by_peer: *by_peer,
             },
-            crate::Event::FileUploadPaused { .. } => return None,
-            crate::Event::FileDownloadPaused { .. } => return None,
+            _ => return None,
```

Comment:
IMO wildcard matcher might bite back if we would implement a new event, the compiler wouldn't point to this place that it needs to be handled